### PR TITLE
Bump to 1.0.2

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/python-pptx LGTM

I also took the freedom to switch the build backend to rattler-build (2x faster build). Let me know what you think.

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+context:
+  version: 1.0.2
+  python_min: "3.9" # remove this line once https://github.com/prefix-dev/rattler-build/pull/1122 is available
+
+package:
+  name: python-pptx
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/python-pptx/python_pptx-1.0.2.tar.gz
```
Comment (synapticarbors) [inline recipe/recipe.yaml]: Should the version be hardcoded here?

File: recipe/recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - lxml >=3.1.0
+    - pillow >=3.3.2
+    - xlsxwriter >=0.5.7
+    - libiconv
+    - typing-extensions
+
+tests:
```
Comment (synapticarbors) [inline recipe/recipe.yaml:31]: Does the test section need a requirements on the python_min as in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

**Review by synapticarbors — COMMENTED** [review]
Thanks for putting this together. I'm not sure why the regro bot stopped picking up updates. I had a few questions (inline) and also:
- will the regrobot handle updating `recipe.yaml` automatically when there are new releases? 
- did you use `conda-recipe-manager` to generate this?
- Does the linter work correctly with the new-style recipe yet?

File: recipe/recipe.yaml (hunk 1 above)
Comment (hadim) [reply recipe/recipe.yaml]: good catch (leftover from debugging)

File: recipe/recipe.yaml (hunk 2 above)
Comment (hadim) [reply recipe/recipe.yaml:31]: Indeed it might be best.

**Comment by hadim** [comment]
LGTM

**Comment by synapticarbors** [comment]
Do you have any thoughts about the questions posed in https://github.com/conda-forge/python-pptx-feedstock/pull/28#pullrequestreview-2419827267?

**Comment by hadim** [comment]
> will the regrobot handle updating recipe.yaml automatically when there are new releases?

Yes. The former one was broken cuz the URL was using `python-pptx` while the pypi URL is now `python_pptx`.

> did you use conda-recipe-manager to generate this?

Yes with some manual edits.

> Does the linter work correctly with the new-style recipe yet?

Yes to my experience so far. The `python {{ python_min }}.*` lint has been fixed already (https://github.com/conda-forge/conda-smithy/pull/2119).

**Comment by synapticarbors** [comment]
@hadim Thanks for your help with this. I've been following the updates on the new recipe format, but haven't implemented it on any of the feedstocks I maintain yet since I wasn't sure how stable it was in terms of regro, linters, etc.

**Comment by hadim** [comment]
It's still young, but so far, the experience and build speed up is great.

**Pull request merged**